core/lib/instrument.c

```diff
@@ -2712,6 +2712,7 @@ dr_get_os_version(dr_os_version_info_t *info)
     get_os_version_ex(&ver, &sp_major, &sp_minor);
```

Comment:
The api/docs/release.dox changelog message is missing: maybe you planned to add it once NtAllocateVirtualMemoryEx and NtMapViewOfSectionEx support is in?  I would say, add it here in the same diff that raises max_supported_os_version.